# RegulatoryTools
Collection of tools dealing with to regulatory sequencing data sets

## Enhancer & target 
- [JEME](https://github.com/yiplabcuhk/JEME/)

## Regulatory variants
- [cepip](http://jjwanglab.org/cepip/index.html)

## QTL
- [mQTLdb](http://www.mqtldb.org/)
- [FastQTL](https://github.com/francois-a/fastqtl)

## 3D genome
- [3DEpiLoop](https://bitbucket.org/4dnucleome/3depiloop)
- [HiC-Bench](https://github.com/NYU-BFX/hic-bench)
- [HiCExplorer](https://github.com/deeptools/HiCExplorer)
- [hichipper](https://github.com/aryeelab/hichipper)
- [HiCToolsCompare](https://bitbucket.org/mforcato/hictoolscompare/downloads/)
- [HiCnv](https://github.com/ay-lab/HiCnv)
- [HiCtrans](https://github.com/ay-lab/HiCtrans)


## Cis‚ÄêrSNPs--Mutation in ChIP-Seq (allele specific binding)
- [GERV](https://github.com/xflicsu/GERV)
- [BaalChIP](https://bioconductor.org/packages/release/bioc/html/BaalChIP.html)
- [AlleleSeq](http://alleleseq.gersteinlab.org/)
- [ABC](https://github.com/mlupien/ABC)

##  allele-specific expression and chromatin modifications (imprinted expression)
- [Allelome.PRO](https://sourceforge.net/projects/allelomepro/)

## allele binding & expression
- [AlleleDB](http://alleledb.gersteinlab.org/download/)

## Motif breaking effect
- [atSNP](https://github.com/chandlerzuo/atSNP)
- [motifBreakR](http://simon-coetzee.github.io/motifBreakR/)
- [FunciVAR](https://github.com/Simon-Coetzee/funcivar)
## Other tools
- [SNPsplit](https://github.com/FelixKrueger/SNPsplit)
## Experience
- [Kai Fu](https://www.pellegrini.mcdb.ucla.edu/pellegrini/pellegrinilabscps/SCP_chipseq.pdf)
